.azure-pipelines.yml

```diff
@@ -1,6 +1,6 @@
 # https://github.com/Microsoft/azure-pipelines-image-generation/blob/master/images/macos/
 variables:
-  MIN_VM_IMAGE: macOS-10.14
+  MIN_VM_IMAGE: macOS-10.15
```

Comment:
please don't do that - it will break the existing min ios version tests (XCode10 is not available for 10.15 image)
Consider playing with https://github.com/appium/WebDriverAgent/blob/877a538cfed0d75f28bb2823e7e7cb8acaa2e358/.azure-pipelines.yml#L303 if you only need to customize integration tests